install.js

```diff
@@ -1,5 +1,16 @@
 const { binWrapper } = require('./index.js');
 
+function sanitizeURL(inputURL) {
+  const parsedURL = new URL(inputURL);
```

Comment:
If `inputURL` is unparseable and an error is thrown, is the default error message detailed enough that the user will know what to do?

Reply:
Check if `inputURL` is indeed required. While thinking about it, I think the best time is before fetching the binary. So I put the check here https://github.com/saucelabs/node-saucectl/pull/28/files#diff-e727e4bdf3657fd1d798edcd6b099d6e092f8573cba266154583a746bba0f346R13